x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -40,17 +40,15 @@ export const DETECTION_RULE_ALERTS_STATUS_API_CURRENT_VERSION = '1';
 export const DETECTION_RULE_RULES_API_CURRENT_VERSION = '2023-10-31';
 
 export const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
-// TODO: REMOVE CSP_LATEST_FINDINGS_DATA_VIEW and replace it with LATEST_FINDINGS_INDEX_PATTERN
-export const CSP_LATEST_FINDINGS_DATA_VIEW = 'logs-cloud_security_posture.findings_latest-*';
+
+export const CDR_LATEST_FINDINGS_INDEX_PATTERN = 'logs-*finding*_latest-*';
```

Comment:
Small detail, just don't want it to get lost:
as per [RFC](https://docs.google.com/document/d/1bEH3AMF_pwk-AYfq-X-2po39ydHT4yEpLT2mamhyP3Y/edit#heading=h.rwfjtkivjn91) the proposed pattern for 3rd party data is `logs-{integration}.{datastream}_latest_misconfigurations_cdr` therefore the pattern to match them is `logs-*_latest_misconfigurations_cdr*` . The reason is that it just happened that wiz data stream is called `cloud-configuration-findings` but we can't rely on that. For querying both our data and 3rd party data the idea was to either have both patterns like `logs-cloud_security_posture.findings_latest-*, logs-*_latest_misconfigurations_cdr` or implement an alias for our current latest index to name it `logs-cloud_security_posture.findings_latest_misconfigurations_cdr` then to be able to catch it with only one index pattern